Ginger/Ginger/BusinessFlowPages_New/ActionsListViewPage.xaml.cs

```diff
@@ -225,14 +225,21 @@ public void UpdateActivity(Activity activity)
         // Drag Drop handlers
         private void listActions_PreviewDragItem(object sender, EventArgs e)
         {
-            if (DragDrop2.DragInfo.DataIsAssignableToType(typeof(Act))
-                || DragDrop2.DragInfo.DataIsAssignableToType(typeof(ApplicationPOMModel))
-                    || DragDrop2.DragInfo.DataIsAssignableToType(typeof(ElementInfo))
-                        || DragDrop2.DragInfo.DataIsAssignableToType(typeof(RepositoryFolder<ApplicationAPIModel>))
-                            || DragDrop2.DragInfo.DataIsAssignableToType(typeof(ApplicationAPIModel)))
+            if (DragDrop2.mDragInfo.DataIsAssignableToType(typeof(Act), true)
+               || DragDrop2.mDragInfo.DataIsAssignableToType(typeof(ApplicationPOMModel), true)
+                   || DragDrop2.mDragInfo.DataIsAssignableToType(typeof(ElementInfo), true)
+                       || DragDrop2.mDragInfo.DataIsAssignableToType(typeof(RepositoryFolder<ApplicationAPIModel>), true)
+                           || DragDrop2.mDragInfo.DataIsAssignableToType(typeof(ApplicationAPIModel), true))
             {
                 // OK to drop
-                DragDrop2.SetDragIcon(true);
+                if (DragDrop2.mDragInfo.Data is ObservableList<RepositoryItemBase>)
```

Comment:
so here we sure that the ObservableList<RepositoryItemBase> contains supported item types like Act/ApplicationModel/ElementInfo/etc...?

Reply:
Yes, I've updated the function "DataIsAssignableToType" to support the same.

Reply:
but here we just check if the data type is ObservableList<RepositoryItemBase> and if yes then we allow to drop, it also can be list of Agents/Envs, no?

Reply:
No. I've modified the existing function "DataIsAssignableToType" and pass the type that supports multiple items drop like in the above line we call the function as : DragDrop2.mDragInfo.DataIsAssignableToType(typeof(Act), true) so inside that function, it'll check if the Object is ObservableList of type Act.